scripts/generate_usb_vif/generate_vif.py

```diff
@@ -129,9 +99,10 @@ def parse_and_add_sink_pdo_to_xml(xml_ele, pdo_value, pdos_info):
         voltage = (pdo_value >> 10) & 0x3ff
         voltage_mv = voltage * 50
         power_mw = (current_ma * voltage_mv) // 1000
-        pdos_info[constants.HIGHER_CAPABILITY_SET] = pdo_value & (1 << 28)
-        pdos_info[constants.FR_SWAP_REQD_TYPE_C_CURRENT_AS_INITIAL_SOURCE] = pdo_value & (3 << 23)
-        add_element_to_xml(xml_ele, constants.SINK_PDO_VOLTAGE, f'{voltage_mv} mV',
+        if pdo_value & (1 << 28):
```

Comment:
Added the comment in new revision

Reply:
The PD spec version is similar to what USB-C subsystem is using. Table number is changing with PD version, but the higher capability bit is same in different versions. When ever there is change is PD revision in USB-C subsystem, we need to remember this spot and update it. Is it fine adding version and table numbers here then.

Reply:
> When ever there is change is PD revision in USB-C subsystem, we need to remember this spot and update it.

This is what I'm trying to avoid. If you put the spec version here, someone will be able to find the definitive reference for this code as long as the previous spec versions are available. Without the spec version, they'll have to hope that the table hasn't changed since you wrote this code.

You should only need to update the version listed here if you make changes to the code that depend on spec content that is newer than the current spec version. Otherwise, the current spec version should be an acceptable reference no matter what we do with the USB-C subsystem.

This is still just a nit.

Reply:
Added revision, versions and table no in new revision

Reply:
PD 2.0 is quite old at this point. Also, I don't think it has APDOs. You should reference PD rev 3.1, v 1.6, table 6-16 here. Battery is 6-18. Variable is 6-17. Augmented is 6-20.

Reply:
I updated the version in new revision. Thank you

Reply:
Good job double-checking my mistake 2ith 6-20.